packages/api/lambdas/reports/granule-inventory-report.js

```diff
@@ -38,59 +40,63 @@ async function createGranuleInventoryReport(recReportParams) {
   ];
 
   const { reportKey, systemBucket } = recReportParams;
-  const searchParams = convertToDBGranuleSearchParams(recReportParams);
-
-  const granulesSearchQuery = getGranulesByApiPropertiesQuery(
-    recReportParams.knex,
-    searchParams,
-    ['collectionName', 'collectionVersion', 'granule_id']
-  );
-  const pgGranulesSearchClient = new QuerySearchClient(
-    granulesSearchQuery,
-    100 // arbitrary limit on how items are fetched at once
-  );
+  try {
```

Comment:
same comment, why try/catch error if we don't need to handle the error?

Reply:
Yeah the main benefit would be logging where the error occurs but I can remove them too

Reply:
https://github.com/nasa/cumulus/pull/3332/commits/ff7b1d42578019db98a58d7ebc1032203f7e853b#diff-3cdf7146169c9c58cfbfd0e1652f49367ffcafafb2527f1979f5458fb77451e0R43